_includes/pbc/all/install-features/202304.0/install-the-inventory-management-feature.md

```diff
@@ -0,0 +1,725 @@
+
+
+This document describes how to ingrate the [Inventory Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html) feature into a Spryker project.
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place.
+
+The current feature integration guide adds the following functionality:
+* [Warehouse User Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html)
```

Comment:
@sakharova-yuliia  I changed the name: `Warehouse Management` => `Warehouse User Management` - this is what it's called in the epic and that's the name of an IG that integrates Warehouse User Management. Correct me if I'm wrong.

Reply:
The epic that we are adding here is called `Warehouse Assignment to Sales Order Items`. It is about assigning warehouse to order item and has nothing to do with Warehouse User. We have a separate epic about Warehouse user but it is a topic of another IG. So I think that we have to keep the previous name` Warehouse Management`